app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,54 @@
+*{
+  padding: 0;
+  margin: 0;
+}
+
+#container {
+  margin: 48px auto;
+  width: 960px;
+
+  section {
+    clear: both;
+
+    p {
+      font-size: 22px;
+    }
+  }
+
+  header {
+    margin-bottom: 46px;
+
+    &.clearfix:after {
+      content: "";
+      display: block;
+      clear: both;
+      visibility: hidden;
+      line-height: 0;
+      height: 0;
+    }
+
+    h1 {
+      float: left;
+      margin: 0;
+    }
+  }
+}
+
+.navigation-links {
+  float: right;
+
+  li {
+    display: inline-block;
+    margin: 0 16px;
+  }
+}
+
+// MISC
+
+html[xmlns].clearfix {
+  display :block;
```

Comment:
Colon after property should be followed by one space<br>Property name should be immediately followed by a colon